framework/include/timeintegrators/AStableDirk4.h

```diff
@@ -62,6 +62,7 @@ class AStableDirk4 : public TimeIntegrator
                                 ADReal & ad_u_dotdot) const override;
   virtual void solve() override;
   virtual void postResidual(NumericVector<Number> & residual) override;
+  virtual bool overridesSolve() const override { return true; }
```

Comment:
Another non-required change, but I think we do this more now via a private param in the base class

Reply:
A private data member or private method? I want the pure virtual to ensure that developers must override it. I initially missed a time integrator in the solid mechanics module which led to a bunch of diffs when I didn't have the pure virtual

Reply:
I didn't realize it was a pure virtual. I like that more. Sounds good to me.